projects/plugins/crm/composer.json

```diff
@@ -16,7 +16,7 @@
 			"pnpm run build"
 		],
 		"build-production": [
-			"pnpm run build-production-concurrently"
+			"pnpm run build"
```

Comment:
As this is now, your production build won't actually set the `NODE_ENV=production` and `BABEL_ENV=production` environment variables for webpack to know to do a production build. You could prefix the command here, or you could make a "build-production" script in package.json, either way works.

Reply:
Gotcha.